ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue

```diff
@@ -309,16 +322,36 @@ export default defineComponent({
     }
 
     const localState = reactive({
+      group: getGroupForOwner(
+        props.editHomeOwner?.id
+      ) as MhrRegistrationHomeOwnerGroupIF,
       ownersGroupId: computed(() =>
-        showGroups.value
-          ? getGroupForOwner(props.editHomeOwner?.id)?.groupId
-          : null
+        showGroups.value ? localState.group?.groupId : null
       ),
       owner: { ...defaultHomeOwner },
       ownerGroupId: undefined,
       showGroups: showGroups,
       isPerson: props.isHomeOwnerPerson,
       isAddingHomeOwner: props.editHomeOwner == null,
+      fraction: {
+        type: '',
+        interest: '',
+        interestNumerator: null,
+        interestTotal: null,
+        tenancySpecified: null
+      } as MhrRegistrationFractionalOwnershipIF,
+      fractionalData: computed(() => {
+        // TODO: get fractional ownership for the group that either owner belongs to or
+        // the group that is selected from dropdown
```

Comment:
What does this mean exactly? 
Is this still a TODO item even though you are getting the `group` and applying it to localstate above?

Reply:
What needs to happen here, if we have multiple groups created, and we select any of them, we should pull the fractions ownership from that group and display it (readonly). If we select a group from dropdown that has not been created yet, than we should display input fields.

Reply:
Fair enough!
Is this work for a future ticket/future PR?

Reply:
This is quite tricky, so it will come in future PR, hence the comment for myself to do it.

Reply:
Sounds good, thanks for the info!